app/domain/search.rb

```diff
@@ -38,13 +44,28 @@ def page=(value)
 
   def sort=(identifier)
     query.sort = identifier
+    raise_if_unrecognised_sort
```

Comment:
As discussed this was done in preparation to extract the `Sort`ing responsabilities into objects in the same way I have done for Filtering. The validation will be placed within the `Sort` object (see [Filter](https://github.com/alphagov/content-performance-manager/blob/d089ef2701d154735d4f4396f8c816fdcefce817/app/domain/search/audit_filter.rb#L14-L14))